### DISCUSSION TITLE
Meetings

### DISCUSSION THREAD

**Discussion by nomaster** [ffddorf/k3os-on-proxmox#2] [General]
Now that #rC3 is over, let's bring our own Adventure Server online!
Düsseldorf could make wonderful use of an open instance to meet, collaborate and party online in a virtual environment. Imagine cool pixel versions of the different places in Düsseldorf!
It should be doable for a small number of people to run and maintain the server. But all of this wouldn't be fun without the artwork.
Chaosdorf already has a pixel map of its own, other organizations and groups might just need some little encouragement or maybe skilled people (hobby artists) are willing to help out?
I propose an open video meeting to get things started and again at a later point in time to see how they progress.

Meeting notes will be taken and published in the wiki. **For details and meeting schedule, [click here](https://github.com/ffddorf/dorfadventure/wiki#meetings)**
Who wants to join?

**Comment by nomaster** [comment]
I have added a meeting link to the post. See you soon!

**Comment by nomaster** [comment]
[Meeting notes are here](https://github.com/ffddorf/dorfadventure/wiki/Meeting-2020-01-02)

**Comment by nomaster** [comment]
Next meeting is today, same time, same place!

**Reply by basisbit** [reply]
Sorry, I will not be able to attend today. Too many meetings at the same time today :-\

**Comment by nomaster** [comment]
There should be a meeting today, so I'm there now.

**Reply by nomaster** [reply]
Meeting notes are [here](https://github.com/ffddorf/dorfadventure/wiki/Meeting-2020-01-16)

**Comment by nomaster** [comment]
**We are moving the meetings to Wednesday 18:00 CET**

**Comment by basisbit** [comment]
Meeting notes from today's meeting can be found [here](https://github.com/ffddorf/dorfadventure/wiki/Meeting-2021-01-27).

**Reply by nomaster** [reply]
did we somehow have a netsplit? Like, two meetings in different rooms at the same time? I have been there for two hours talking to people, but I didn't see you and the meeting notes are unfamiliar.

**Comment by AnsisMalins** [comment]
Looks like it. We were in ExtremeClocksPrecedeTightly.

**Reply by nomaster** [reply]
That's funny. How come you changed the room? Did I miss anything?

**Reply by AnsisMalins** [reply]
It's the same room we used last week.

**Reply by AnsisMalins** [reply]
Mentioned here: https://github.com/orgs/ffddorf/teams/dorfadventure/discussions/2?from_comment=7#discussion-2-comment-7

**Reply by AnsisMalins** [reply]
I'll take this as an opportunity to suggest we name our meeting someting sensible, like dorfadventure.

**Reply by nomaster** [reply]
Yes we can. But I'm a bit disappointed, since the meeting link is here in the original message. I'll take care.

**Comment by nomaster** [comment]
The meeting URL has been changed. For future meetings, please join here: https://jitsi.freifunk-duesseldorf.de/DorfAdventure

**Comment by nomaster** [comment]
This weeks meeting notes are [here](https://github.com/ffddorf/dorfadventure/wiki/Meeting-2021-02-03)

**Comment by nomaster** [comment]
We decided to meet 30 minutes later from now on. New meeting schedule is **Wednesday, 18:30 CET**

**Comment by nomaster** [comment]
This week's meeting notes are [here](https://github.com/ffddorf/dorfadventure/wiki/Meeting-2021-02-10)

**Comment by basisbit** [comment]
Last week's meeting notes are [here](https://github.com/ffddorf/dorfadventure/wiki/Meeting-2021-02-17)

**Comment by nomaster** [comment]
This week's meeting notes are [here](https://github.com/ffddorf/dorfadventure/wiki/Meeting-2021-02-24)

**Comment by nomaster** [comment]
This week's meeting notes are short, but they are [here](https://github.com/ffddorf/dorfadventure/wiki/Meeting-2021-03-03)

**Comment by nomaster** [comment]
This week's meeting notes are [here](https://github.com/ffddorf/dorfadventure/wiki/Meeting-2021-03-10)

**Comment by nomaster** [comment]
This week's meeting notes are [here](https://github.com/ffddorf/dorfadventure/wiki/Meeting-Notes-2021-03-17)

**Comment by nomaster** [comment]
This week's meeting notes are [here](https://github.com/ffddorf/dorfadventure/wiki/Meeting-2021-03-24)

**Comment by nomaster** [comment]
This week's meeting notes are [here](https://github.com/ffddorf/dorfadventure/wiki/Meeting-2021-03-31)

**Comment by nomaster** [comment]
This week's meeting notes are [here](https://github.com/ffddorf/dorfadventure/wiki/Meeting-2020-04-07)

**Comment by nomaster** [comment]
This week's meeting notes are [here](https://github.com/ffddorf/k3os-on-proxmox/wiki/Meeting-2021-04-14)

**Comment by nomaster** [comment]
This week's meeting notes are [here](https://github.com/ffddorf/k3os-on-proxmox/wiki/Meeting-2021-04-21)

**Comment by nomaster** [comment]
This week's meeting notes are [here](https://github.com/ffddorf/k3os-on-proxmox/wiki/Meeting-2021-04-28)

**Comment by nomaster** [comment]
This week's meeting notes are [here](https://github.com/ffddorf/k3os-on-proxmox/wiki/Meeting-2021-05-05)

**Comment by nomaster** [comment]
This week's meeting notes are [here](https://github.com/ffddorf/k3os-on-proxmox/wiki/Meeting-2021-05-19)
